.github/workflows/helm-chart-releaser.yml

```diff
@@ -29,8 +29,10 @@ jobs:
       with:
         version: 'v3.6.0' # default is latest stable
       id: install
-    - name: validate
+    - name: validate meshery
       run: helm lint install/kubernetes/helm/meshery --with-subcharts
+    - name: validate meshery-operator
```

Comment:
Due to https://github.com/stefanprodan/helm-gh-pages/issues/23#issue-800058519, I'd like to make little change on the helm-charts CI.